dev-system/src/cmr/dev_system/system.clj

```diff
@@ -117,7 +117,7 @@
 
 (defmethod create-elastic :external
   [_]
-  (elastic-config/set-elastic-port! 9209))
```

Comment:
so we are no longer supporting external elastic? If so, then should we just get rid of this method entirely?

Reply:
We can use external elastic just fine, it just doesn't seem there's a real need to set the port to 9209 when you can just use 9210 which is the default that CMR checks for elasticsearch on